components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc

```diff
@@ -161,23 +164,27 @@ Consequently, the fee associated with a single L2→L1 message is:
 \text{gas_price}\cdot c_w\cdot(3+\text{payload_size})
 ++++
 
-where stem:[$c_w$] is the calldata cost (in gas) per 32-byte word.
+where stem:[$c_w$] is the calldata cost, in gas, per 32-byte word.
 
 ==== Deployed contracts
 
 When a transaction that raises the `deploy` syscall is included in a state update, the following data reaches L1:
 
-* Contract address
-* Class hash
+* contract address
+* new nonce
+* number of storage updates
```

Comment:
of storage updates and new nonce are a single element, so one bulletpoint

Reply:
Fixed.